.github/workflows/refresh-doc.yml

```diff
@@ -0,0 +1,75 @@
+name: refresh-doc
+on:
+  push:
+    branches:
+      - hannah-github-actions
+
+env:
+  WEB_URL: 'https://cold-pigs-fail-174-94-59-90.loca.lt' # This needs to be updated to the live web server!
```

Comment:
We can use something like https://requestbin.net/r/b9atr5pk?inspect temporarily until the endpoint is up.

Reply:
Totally! Could you help me set this up?

Reply:
Will do!